torchvision/prototype/transforms/_presets.py

```diff
@@ -97,3 +98,36 @@ def forward(self, img: Tensor, target: Optional[Tensor] = None) -> Tuple[Tensor,
                 target = F.pil_to_tensor(target)
             target = target.squeeze(0).to(torch.int64)
         return img, target
+
+
+class RaftEval(nn.Module):
+    def forward(
+        self, img1: Tensor, img2: Tensor, flow: Optional[Tensor], valid_flow_mask: Optional[Tensor]
+    ) -> Tuple[Tensor, Tensor, Optional[Tensor], Optional[Tensor]]:
+
+        img1, img2, flow, valid_flow_mask = self._pil_or_numpy_to_tensor(img1, img2, flow, valid_flow_mask)
+
+        img1 = F.convert_image_dtype(img1, torch.float32)
+        img2 = F.convert_image_dtype(img2, torch.float32)
+
+        # map [0, 1] into [-1, 1]
+        img1 = F.normalize(img1, mean=0.5, std=0.5)
+        img2 = F.normalize(img2, mean=0.5, std=0.5)
+
+        img1 = img1.contiguous()
+        img2 = img2.contiguous()
+
+        return img1, img2, flow, valid_flow_mask
+
+    def _pil_or_numpy_to_tensor(self, img1, img2, flow, valid_flow_mask):
```

Comment:
Typing for JIT script:
```suggestion
    def _pil_or_numpy_to_tensor(self, img1: Tensor, img2: Tensor, flow: Optional[Tensor], valid_flow_mask: Optional[Tensor])-> Tuple[Tensor, Tensor, Optional[Tensor], Optional[Tensor]]:
```

Or you can also decide to move this private method in the main body of forward since you use it once, to avoid defining typing multiple times. Up to you.